b2share/modules/oaiserver/cli.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2016 University of Tuebingen, CERN, SurfSara
+#
+# B2Share is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# B2Share is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with B2Share; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""B2Share OAI-PMH cli commands."""
+
+
+from __future__ import absolute_import, print_function
+
+import click
+from flask_cli import with_appcontext
+from sqlalchemy.orm.attributes import flag_modified
+
+from invenio_db import db
+from invenio_oaiserver.models import OAISet
+from invenio_records.models import RecordMetadata
+from invenio_records_files.api import Record
+
+from b2share.modules.communities.api import Community
+from b2share.modules.records.utils import list_db_published_records
+
+
+
+@click.group()
+def oai():
+    """OAI commands."""
+
+
+@oai.command()
+@with_appcontext
+def update_sets():
```

Comment:
we could add a fixme, for removing those commands once the b2share.eudat.eu is migrated.